### ISSUE TITLE
adding more chord types

### ISSUE THREAD

**Issue by dgy18787** [rousselmanu/msc_plugins#2]
Could I add more chord types in the plugins like add9 or sus4?

**Comment by rousselmanu** [comment]
Yes you can modify the [structure here](https://github.com/rousselmanu/msc_plugins/blob/master/chordIdentifier.qml#L140) where you could add the intervals corresponding to the chords to be identified.

**Comment by dgy18787** [comment]
I have tried to add some new chords like [5,7] in this array but the plugins seems to be unable to mark it right (sus4). Should I also defined its name in chord_str array?
